recipes/jest-to-node-test-runner/src/workflow.test.ts

```diff
@@ -0,0 +1,20 @@
+import { resolve } from "node:path";
+import { execPath } from "node:process";
+import { readFile } from "node:fs/promises";
+import { describe, it, type TestContext } from "node:test";
+import { fileURLToPath } from "node:url";
+
+import { spawnPromisified } from "../test/spawn-promisified.ts";
+import { workflow } from "./workflow.ts";
+
+describe("workflow", () => {
+	it("should update bad specifiers and ignore good ones", async (t: TestContext) => {
```

Comment:
no needed to explicitly write this type because it's provide by type def of `it` function 

```suggestion
	it("should update bad specifiers and ignore good ones", async (t) => {
```

Reply:
TS will complain when I use strict assertions. Maybe it is because now they're legacy and I can use `deepEqual` instead

Reply:
I think something is wrong then: you should indeed not have to add the type (I haven't seen it complain before 🤔)